test/integration/apiserver/discovery/discovery_test.go

```diff
@@ -52,9 +52,16 @@ import (
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
+//lint:ignore U1000 we need to alias only for the sake of embedding
```

Comment:
we should report this to the linter, this is a false positive (since the type is clearly used below)